formats/ome-converter-plugin/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM polusai/bfio:2.1.9
+FROM polusai/bfio:2.3.2
```

Comment:
One last update is required
change base container image from FROM polusai/bfio:2.3.2 to FROM polusai/bfio:2.3.3

Reply:
Done